cookbooks/bcpc-hadoop/recipes/disks.rb

```diff
@@ -125,10 +125,15 @@
       fs_check.run_command
 
       dev_name = if fs_check.status.success? &&
-                     fs_check.stdout.include?('SGI XFS filesystem')
+                    fs_check.stdout.include?('SGI XFS filesystem')
                    "/dev/#{base_name}"
                  else
-                   "/dev/#{base_name}1"
+                   case base_name
+                   when /^nvme/
+                     "/dev/#{base_name}p1"
```

Comment:
Where are you partitioning the NVMe drives?

Reply:
https://github.com/bloomberg/chef-bach/blob/c84457b9b44242c0eb172c99b750d683dfa000f5/cookbooks/bcpc-hadoop/recipes/disks.rb#L149-L150

Reply:
Thanks but that's after you ascertained there's a partition (`p1`) on that disk right?